src/test/tcp/test_tcp.c

```diff
@@ -759,6 +814,7 @@ int main(int argc, char *argv[]) {
 
     iowait_func wait = NULL;
     int use_iov = 0;
+    char *queuename = QUEUENAME;
```

Comment:
We should try to remove the queue at the end, using `msgctl`. This should probably happen in an `atexit` handler so that it still happens on test failures.

Also, slight preference for getting the queue handle here and passing the handle around rather than passing the string around. That keeps the queue management at roughly a single (outer) scope, and lets us pass around a more strongly-typed handle rather than just a string.